recipes/mdns/all/test_package/test_package.c

```diff
@@ -0,0 +1,363 @@
+
```

Comment:
could you please keep it shorter? You can use only a method (e.g. mdns_socket_open_ipv4) and remove all those user arguments.